homeassistant/components/limitlessled/light.py

```diff
@@ -285,29 +287,27 @@ def max_mireds(self):
         """Return the warmest color_temp that this light supports."""
         return 370
 
+    @property
+    def color_mode(self) -> str | None:
+        """Return the color mode of the light."""
+        if self._fixed_color_mode:
+            return self._fixed_color_mode
+
+        # The light supports both hs and white with adjustable color temperature
+        if self._effect == EFFECT_NIGHT or self._color is None or self._color[1] == 0:
```

Comment:
This is copying the condition from the `hs_color` attribute, which was previously gated by checking if the saturation was 0. @smwa can you help confirm that's the correct way to determine the current mode of the light (color or white with adjustable color temperature)?